torch/jit/__init__.py

```diff
@@ -1123,7 +1125,11 @@ def __init__(self, original, stubs):
             # Copy Parameters / Modules / Buffers
             for name in dir(original):
                 item = getattr(original, name)
-                if isinstance(item, Parameter) or (isinstance(item, Module) and item is not self):
+                if item is None:
```

Comment:
shouldn't this also check that name is in the parameter list? Otherwise you are copying over a non-parameter attribute that would otherwise have not been copied.